docs/CARS.md

```diff
@@ -1,185 +1,200 @@
+| Toyota  | Prius Prime 2017-20         |        All        |     ⭐<sup>1</sup>      |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Toyota  | Prius Prime 2021-22         |        All        |           ⭐            |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Toyota  | RAV4 2019-21                |        All        |           ⭐            |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Toyota  | RAV4 Hybrid 2016-18         |       TSS-P       |     ⭐<sup>1</sup>      |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Toyota  | RAV4 Hybrid 2019-21         |        All        |           ⭐            |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Toyota  | Sienna 2018-20              |        All        |     ⭐<sup>1</sup>      |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+
+## Silver Cars
+
+| Make       | Model (US Market Reference)     | Supported Package | openpilot Longitudinal | Longitudinal to 0mph | Steering to 0mph | Good steering torque | Well Supported |
+|------------|---------------------------------|:-----------------:|:----------------------:|:--------------------:|:----------------:|:--------------------:|:--------------:|
+| Audi       | A3 2014-19                      | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Audi       | A3 Sportback e-tron 2017-18     | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Audi       | Q2 2018                         | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Audi       | Q3 2020-21                      | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Audi       | S3 2015                         | ACC + Lane Assist |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Genesis    | G70 2018                        |        All        |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
+| Genesis    | G70 2020                        |        All        |     ⭐<sup>3</sup>      |          ⭐           |        ⭐         |          ⭐           |                |
+| Genesis    | G90 2018                        |        All        |     ⭐<sup>3</sup>      |          ⭐           |        ⭐         |                      |       ⭐        |
+| Hyundai    | Ioniq Electric 2020             |    SCC + LKAS     |                        |          ⭐           |        ⭐         |          ⭐           |       ⭐        |
```

Comment:
<s>All HKG should have long too</s>, only counting those with radar points for now